tests/ai_safety/evalhub/evalcard/conftest.py

```
@@ -0,0 +1,316 @@
+    ) as mlflow:
+        mlflow_deployment = Deployment(
+            client=admin_client,
+            name="mlflow",
+            namespace=py_config["applications_namespace"],
+        )
+        mlflow_deployment.wait_for_replicas(timeout=300)
+        yield mlflow
+
+
+@pytest.fixture(scope="class")
+def evalhub_mt_cr(  # noqa: UFN001
```

Ruff (0.15.21)

[warning] Invalid rule code in `# noqa`: UFN001

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)